docs/module-list.md

```diff
@@ -0,0 +1,44 @@
+# Module list
+
+This is a list of existing dfTimewolf modules. To see how well they play
+together, see the [recipe list](recipe-list.md).
+
+## Collectors
+
+* `FilesystemCollector` - a simple collector that just passes a local path on to
+the processors.
+
+### GRR hunts
+
+Launch or fetch results from fleet-wide GRR hunts.
+  * `GRRHuntArtifactCollector` - Launches a fleet-wide GRR
+  `ArtifactCollectorFlow`
```

Comment:
I think you forgot to add a description here

Reply:
I meant to write "Launches a fleet-wide GRR `ArtifactCollectorFlow`" on the same line.